css-canvas-medical/index.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Canvas Medical</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="nav">
+      <div class="column-header">
+        <img src="./images/logo-blue.svg" class="logo">
+      </div>
+      <div class="column-header">
+        <button class="btn">Product</button>
+        <button class="btn">Implementation</button>
+        <button class="btn">About Us</button>
+        <button class="btn">Blog</button>
+        <button class="btn" id="blue">Contact Us</button>
+      </div>
+    </div>
+    <div class="row">
+      <div class="overlay"></div>
+      <div class="img-text">
+        <h1 class="img-head">The EMR Reimagined for Primary Care</h1>
+        <p class="text">Can't remember the last time you talked with a patient without  focusing
+          on the EMR? Canvas lets you rediscover joy in the work you do. Capture meaningful data, identify gaps in care,
+          seamlessly between tasks within the platform. You stay present with patients
+          while your whole care team stays up-to-speed. Be a better doctor with Canvas, designed for primary care providers with the
+          help of primary care providers.
+        </p>
+        <button class="start-button">Get Started</button>
+      </div>
+      <img src="./images/hero.jpg" alt="hero" id="hero">
+    </div>
+    <div class="row-mid">
+      <div class="column-mid">
+        <h1 class="mid-header">Clinitians Prefer Canvas</h1>
+        <p class="small-font">Physician-owned primary care practices across the country have already
+         made Canvas the fabric of their care
+        </p>
+      </div>
+    </div>
+    <div class="row-addresses">
```

Comment:
You shouldn't have individually named `row` classes. You should only have one `row` class with `display: flex` as its only styling, and any additional styling necessary should be applied to other classes you can add to the element in addition to the `row` class.